modules/nw-egress-ips-config-object.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
+
+[id="nw-egress-ips-config-object_{context}"]
+= EgressIPconfig object
+As a feature of egress IP, the `reachabilityTotalTimeoutSeconds` configures the total timeout for checks that are sent by probes to egress IP nodes. The`egressIPConfig` object allows users to set the `reachabilityTotalTimeoutSeconds` `spec`. If the EgressIP node cannot be reached within this timeout, the node is declared down.
+
+You can increase this value if your network is not stable enough to handle the current default value of 1 second.
+
+The following YAML describes changing the `reachabilityTotalTimeoutSeconds` from the default 1 second probes to 5 second probes:
+
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressIP
+  name: networks.operator.openshift.io
+  spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  defaultNetwork:
+    ovnKubernetesConfig:
+      egressIPConfig: <1>
+        reachabilityTotalTimeoutSeconds: 5 <2>
+      gatewayConfig:
+        routingViaHost: false
+      genevePort: 6081
+----
+<1> The `egressIPConfig` holds the configurations for the options of the `EgressIP` object. Changing these configurations allows you to extend the `EgressIP` network object.
```

Comment:
Are the _the `EgressIP` object_ and _the `EgressIP` network object_ the same thing?